CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 ## Unreleased
+- Add Elasticsearch highlighting match option for Simplequery model
```

Comment:
- Add the option to highlight the part that matched the use case for simplequery models